packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,55 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Align content within divider
+ * @public
+ */
+export const AlignContent = {
+  /**
+   * Default. Center align content.
+   */
+  center: 'center',
+  /**
+   * Align content at flex start
+   */
+  start: 'start',
+  /**
+   * Align content at flex end
+   */
```

Comment:
These are pretty intuitive IMO. We can add comments for each but I'm not sure what will get documented, stripped out, etc...I'd probably remove for the values of the const object at this point. We can take this as a discussion point though if need be.

Reply:
I'm fine removing comments. These properties are very easy to understand. To be sure, if I were including a more complex set of properties, I would document them as I had.